src/TFR.jl

```diff
@@ -165,15 +165,15 @@ function istft(stft::Array{Complex{T}, 3},
                                    windowsize::Int = 2 * (size(stft, 1) - 1),
                                    hopsize::Int = windowsize >> 2; kwargs...) where {T <: AbstractFloat}
     nchannels = size(stft, 3)
-    buffers = cell(nchannels)
+    buffers = Vector{Any}(undef, nchannels)
```

Comment:
`Vector{Any}` is type unstable. Why don't you know here what elements this will contain?